@here/harp-mapview/lib/VisibleTileSet.ts

```diff
@@ -837,6 +844,32 @@ export class VisibleTileSet {
         tile.dispose();
     }
 
+    /**
+     * Disposes of tiles that overlap twice.
+     **/
+    private checkDuplicateFullyCoveringTiles(dataSource: DataSource, tile: Tile) {
+        if (dataSource.isFullyCovering && tile.backgroundPlane !== undefined) {
```

Comment:
Why checking `tile.backgroundPlane !== undefined`? For the WebTile datasource there is no backgroundPlane but the tiles are still fully covered

Reply:
I wasn't considering the webtile datasource, thanks for the reminder.

I use the backgroundPlane render order to determine which tile should be removed. If it doesn't exist, I will take it to mean that it should have the highest render order, and all existing / future tiles should be removed.